ansys/dpf/core/field.py

```diff
@@ -262,12 +264,20 @@ def plot(self, shell_layers=None, **kwargs):
         shell_layers : shell_layers, optional
             Enum used to set the shell layers if the model to plot
             contains shell elements. The default is ``None``.
+        scaling_result : result operator, optional
```

Comment:
This class name will reflect poorly in the doc, the real type name needs to be used.
I also think that restricting the input type to "Result" is not a good idea, Operators (maybe even Field and Fields Containers) should be supported. "Result" is a helper but not a main DPF entity